comonad.cabal

```
@@ -68,6 +68,17 @@ flag distributive
   default: True
   manual: True
 
+flag indexed-traversable
+  description:
+    You can disable the use of the `indexed-traversable` package using `-f-indexed-traversable`.
+    .
+    Disabling this is an unsupported configuration, but it may be useful for accelerating builds in sandboxes for expert users.
+    .
+    If disabled we will not supply instances of `indexed-traversable`
```

Since the description for the `-fdistributive` flag mentions the `Distributive` class, I think the description for `-findexed-traversable` should be analogous:

```suggestion
    If disabled we will not supply instances of `FunctorWithIndex`
```